assets/js/components/settings/SettingsActiveModule/index.test.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * SettingsModule component tests.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { createHashHistory } from 'history';
+import { Switch, Route } from 'react-router-dom';
+
+/**
+ * Internal dependencies
+ */
+import SettingsActiveModule from '.';
+import { render, fireEvent, createTestRegistry, provideModules } from '../../../../../tests/js/test-utils';
+
+describe( 'SettingsModule', () => {
+	// const mockAnalyticsModule = {
+	// 	active: true,
+	// 	autoActivate: true,
+	// 	dependantModulesText: '',
+	// 	description: 'Analytics test module description',
+	// 	homepage: '#',
+	// 	name: 'Analytics',
+	// 	provides: [ 'foo', 'bar' ],
+	// 	setupComplete: false,
+	// 	slug: 'analytics',
+	// };
+
+	const SettingsModuleWithWrapper = () => (
+		<Switch>
+			<Route path={ [ '/connected-services/:moduleSlug/:action', '/connected-services/:moduleSlug', '/connected-services' ] }>
+				<SettingsActiveModule slug="analytics" />
+			</Route>
```

Comment:
Can we avoid duplicating the routes here and use a "catch-all" route instead? It seems like it will always render analytics anyways so I would think the `:moduleSlug` part isn't needed anyways 🤔

Reply:
What do you mean by a "catch-all" route? I don't quite think I follow 😅

Reply:
Sorry, I think I misunderstood. It looked like we were duplicating a route definition here that could render different modules but would always render Analytics regardless. I see that the parameters are significant to the behavior of the component now (via `useParams`), it just wasn't immediately obvious before.